am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/Resource.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class Resource {
+    private final String resourceId;
+    private final ResourceMetadata type;
```

Comment:
Maybe `ResourceDefinition` would be a better name?

#minor.

Reply:
I thought about `ResourceDefinition` as well. Either we change now or later.